.gitignore

```diff
@@ -0,0 +1 @@
+*.pyc
```

Comment:
It's better to make a new .gitignore file and put it under the `server` directory. You can use https://www.gitignore.io/ to make the gitignore file after consulting with the backend team on what tech you guys are using.